cmd/cluster-ingress-operator/main.go

```diff
@@ -73,19 +73,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	operatorConfig := operatorconfig.Config{
+		OperatorReleaseVersion: os.Getenv("RELEASE_VERSION"),
+		Namespace:              operatorNamespace,
+		RouterImage:            routerImage,
+		Platform:               infraConfig.Status.Platform,
```

Comment:
Can we go ahead and remove `Platform` from this struct? I don't think it should ever have been there. Anything that needs an `Infrastructure` should be looking it up or taking an argument at this point.

Reply:
done